LICENSE.md

```diff
@@ -1,4 +1,3 @@
-Copyright (c) Microsoft Corporation
```

Comment:
I think we need this line back in though

Reply:
You rịght! This package is copied from Microsoft so it needs that.